components/jira/actions/create-issue/create-issue.mjs

```diff
@@ -199,8 +218,8 @@ export default {
     let properties;
     try {
       properties = JSON.parse(this.properties);
-    } catch ( err ) {
-      //pass
+    } catch (err) {
```

Comment:
If we fail to parse we just ignore it? There are no side effects of doing this?

Reply:
Yeah, I guess. Added error logging